MODULE.bazel

```diff
@@ -22,212 +22,7 @@ use_repo(
 )
 
 go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
-go_deps.module(
```

Comment:
Is it possible that many of these dependencies aren't needed anymore? Our `go.mod` is much leaner than this list.